api/v1beta1/grafana_types.go

```diff
@@ -119,6 +119,9 @@ type GrafanaClient struct {
 	// TLS Configuration used to talk with the grafana instance.
 	// +optional
 	TLS *TLSConfig `json:"tls,omitempty"`
+	// Optional map of header key and value.
```

Comment:
I would probably clarify the description a bit for end users, something along the lines of "Custom **HTTP** headers"

Reply:
what do you think about the following line : ` Custom HTTP headers to use when interacting with this Grafana.`

Reply:
Should be OK